docs/proposal-rose-suite-run.md

```diff
@@ -18,7 +18,7 @@ functionalities of Rose.
 
 In reality, Cylc has become the centre of the Rose application suites, and the
 extra functionalities provided by Rose are equally desirable on sites who
-choose not to use Rose.
+choose not to use Cylc.
```

Comment:
No, I believe that's meant be "Rose".  That's why we want to migrate certain Rose functionalities into Cylc, so even those who do not use Rose can get the benefits when they use Cylc.

Reply:
Ah, OK, thanks for clarifying. The sentence didn't make sense to me originally after reading through several times (the double reference to Rose), but now I do see what it is getting at.